VS4Mac_Meadow_Extension/MeadowProject.cs

```diff
@@ -22,11 +24,10 @@ protected override void OnEndLoad()
 
             Console.WriteLine("WLABS: OnEndLoad");
 
-            // if the project is not a library
-            // shouldn't this test if it's an executable?
-            if (Project.CompileTarget != CompileTarget.Library)
+            // Check that the project is a library, so we can start listening for attached/detached devices.
+            if (Project.CompileTarget == CompileTarget.Library)
```

Comment:
@adrianstevens, @alexischr should this be a hard rule? Meaning we *only* accept libraries from now on?